WooCommerce/Classes/ViewRelated/Products/ProductsTopBannerFactory.swift

```diff
@@ -70,7 +70,7 @@ private extension ProductsTopBannerFactory {
         static let title = NSLocalizedString("New features available!",
                                              comment: "The title of the top banner on the Products tab.")
 
-        static let infoVariations = NSLocalizedString("You can now create and manage product variations!",
+        static let topBannerInfo = NSLocalizedString("",
```

Comment:
I wonder if is alright to leave this String empty here, as we do not know what feature we'll be promoting, but doesn't hurt to keep the messaging. What do you think?

Reply:
I think leaving this as an empty string is reasonable, since we're not showing the banner now and when we use it again we'll update it with new messaging

To avoid any confusion for translators, I'd suggest we remove the localization entirely. Something like:

```
        static let topBannerInfo = "" // The info of the top banner on the Products tab.
```